### PR TITLE
Add new validator pages.

### PR DESCRIPTION
**Pull request by srinu391**

### CODE REVIEW

File: src/containers/ChainEndpoints/data.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+export const chainEndPoints = [
```
Comment (dheerajkd30) [inline src/containers/ChainEndpoints/data.js:1]: can you verify the contents here?

File: src/containers/IbcRelayers/data.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+export const ibcRelayers = [
+  {
+    key: "1",
+    relayer: "AUDIT.One",
+    email: "hello@audit.one",
```
Comment (dheerajkd30) [inline src/containers/IbcRelayers/data.js:5]: can you verify the contents here?

File: src/containers/Mainnet/data.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      destination_channel: 'channel-30',
+      known_operators: 'Simply Staking *looking for more operators*'
+    },
+    {
+      key: '7',
+      source_channel: 'channel-38',
+      destination_chain: 'Gravity Bridge',
+      destination_chainid: 'gravity-bridge-3',
+      destination_channel: 'channel-24',
+      known_operators: 'AUDIT.one, bitszn, Cosmonaut Stakes, CryptoCrew, Stakin'
```
Comment (dheerajkd30) [inline src/containers/Mainnet/data.js:56]: Same here. Don’t think audit one is relaying

File: src/containers/Testnet/data.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+export const testnetIbcChannels = [
+    {
+      key: '1',
+      source_channel: 'channel-51',
+      destination_chain: 'Axelar',
+      destination_chainid: 'axelar-dojo-1',
+      destination_channel: 'channel-42',
+      known_operators: 'AUDIT.one, Cros-nest, Polkachu *looking for more operators*'
```
Comment (dheerajkd30) [inline src/containers/Testnet/data.js:8]: can you verify the contents here? Audit one !!!

File: src/containers/Testnet/index.js [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+];
+
+const dataSource = [
+  {
+    key: "1",
+    source_channel: "channel-51",
+    destination_chain: "Axelar",
+    destination_chainid: "axelar-dojo-1",
+    destination_channel: "channel-42",
+    known_operators:
```
Comment (dheerajkd30) [inline src/containers/Testnet/index.js:44]: can you verify the contents here !!!!

**Review by dheerajkd30 — CHANGES_REQUESTED** [review]

File: src/containers/IbcRelayers/data.js (hunk 2 above)
Comment (srinu391) [reply src/containers/IbcRelayers/data.js:5]: This data is taken from figma. Can you check and provide me with correct details.

File: src/containers/Mainnet/data.js (hunk 3 above)
Comment (srinu391) [reply src/containers/Mainnet/data.js:56]: Need proper data for same. Data taken from designs

File: src/containers/Testnet/data.js (hunk 4 above)
Comment (srinu391) [reply src/containers/Testnet/data.js:8]: Can you provide the proper data , this taken from designs

File: src/containers/Testnet/index.js (hunk 5 above)
Comment (srinu391) [reply src/containers/Testnet/index.js:44]: Need proper data

**Review by dheerajkd30 — APPROVED** [review]

**Pull request merged**